src/App.tsx

```diff
@@ -1,9 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App: React.FC = () => {
+function prepareMovies(query: string) {
+  return moviesFromServer.filter(movie => {
+    const { title, description } = movie;
+
+    return filterWord(title, query) || filterWord(description, query);
```

Comment:
let's fix it

<img width="519" alt="image" src="https://user-images.githubusercontent.com/86183007/228943462-59a42d15-025c-4d19-afc7-0e16f8a67056.png">